shared/coordinates.js

```diff
@@ -39,15 +39,15 @@ export function splitCoordinate(coordinate) {
  * @return {Object} coordinates  - Object with a x[] and y[] properties
  */
 export function splitCoodinatePairs(pairs) {
-  const x = []
-  const y = []
+  const xs = []
+  const ys = []
 
   for (const pair of pairs) {
-    x.push(pair.x)
-    y.push(pair.y)
+    xs.push(pair.x)
+    ys.push(pair.y)
   }
 
-  return { x, y }
+  return { xs, ys }
```

Comment:
why this rename?

Reply:
I tried to reflect the fact that we're returning an array of each coordinate.
Two things in particular made me made up my mind:
- We often use `splitCoodinatePairs` to send coordinates to contract methods that already receive them as `xs` and `ys`
- In a few places we had to avoid destructuring the result of calling splitCoodinatePairs because x and y were already present in the scope

Reply:
sabe